### PR TITLE
Removes @history and deprecation notices

### PR DESCRIPTION
**Pull request by itamarhaber**
As these were moved into Redis core (pending fixes included in https://github.com/redis/redis/pull/10016).

Related to https://github.com/redis/redis-io/pull/260

### CODE REVIEW

File: commands/scan.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ As you can see most of the calls returned zero elements, but the last call where
 
 ## The TYPE option
 
-As of version 6.0 you can use this option to ask `SCAN` to only return objects that match a given `type`, allowing you to iterate through the database looking for keys of a specific type. The **TYPE** option is only available on the whole-database `SCAN`, not `HSCAN` or `ZSCAN` etc.
+You can use the `!TYPE` option to ask `SCAN` to only return objects that match a given `type`, allowing you to iterate through the database looking for keys of a specific type. The **TYPE** option is only available on the whole-database `SCAN`, not `HSCAN` or `ZSCAN` etc.
```
Comment (oranagra) [inline commands/scan.md:145]: what's with the `!`?

File: commands/georadius.md [hunk 2]
```diff
@@ -58,11 +56,7 @@ Since `GEORADIUS` and `GEORADIUSBYMEMBER` have a `STORE` and `STOREDIST` option
 
 Breaking the compatibility with the past was considered but rejected, at least for Redis 4.0, so instead two read-only variants of the commands were added. They are exactly like the original commands but refuse the `STORE` and `STOREDIST` options. The two variants are called `GEORADIUS_RO` and `GEORADIUSBYMEMBER_RO`, and can safely be used in replicas.
 
-Both commands were introduced in Redis 3.2.10 and Redis 4.0.0 respectively.
-
-@history
-
-* `>= 6.2`: Added the `ANY` option for `COUNT`.
+Both commands were introduced in Redis 3.2.10 and Redis 4.0.0, respectively.
```
Comment (oranagra) [inline commands/georadius.md:59]: shouldn't that be deleted (and make sure it's in the json file?

File: commands/scan.md (hunk 1 above)
Comment (itamarhaber) [reply commands/scan.md:145]: This is in order to "escape" the thing so it doesn't auto-link to the Redis `TYPE` command here.

File: commands/georadius.md (hunk 2 above)
Comment (itamarhaber) [reply commands/georadius.md:59]: Yep - the json is wrong, and this can be deleted.

**Pull request merged**